src/components/RiverConditions_Carousel.vue

```diff
@@ -12,13 +12,17 @@
                 v-for="chart in charts"
```

Comment:
here try using `v-for="(chart, index) in charts` to be able to leverage `index` in the `handleSlideClick` function